projects/lf-documentation/src/app/app.component.html

```diff
@@ -1,5 +1,13 @@
 <mat-toolbar color="primary" class="documentation-font">
   <mat-toolbar-row>
+    <button class="arrow-left">
+      <a href="https://developer.laserfiche.com">
```

Comment:
put this link as a variable in app.component.ts (and add a TODO to update it once we have the landing page